src/helpers/api.ts

```diff
@@ -0,0 +1,161 @@
+import axios, { AxiosError, AxiosRequestConfig, AxiosResponse } from 'axios';
+import store from '@/store/index';
+import apiUrls from '@/setup/apiUrls.json';
+import { IMessage, INotification } from '@/types/c-notification';
+
+interface IUrlKeyValues {
+  [key: string]: string;
+}
+
+export interface IApi {
+  getUrl: (urlKey: keyof typeof apiUrls, values: IUrlKeyValues) => string;
+  // eslint-disable-next-line max-len, @typescript-eslint/no-explicit-any
```

Comment:
Should we modify the max-len rule in a way that it allows longer lines that contain TS definitions?